src/ngraph/runtime/gpu/gpu_emitter.cpp

```diff
@@ -739,26 +668,27 @@ void runtime::gpu::GPU_Emitter::emit_Max(EMIT_ARGS)
         // one of args0 axes has zero size, zero output, use args1 value
         if (args[0].get_size() == 0)
         {
-            writer << out[0].get_type()
-                   << " init_value = " << TypeInfo::Get(args[0].get_type())->min() << ";\n";
-            writer << "vector<" << out[0].get_type() << "> temp(" << out[0].get_size()
-                   << ", init_value);\n";
-            writer << "runtime::gpu::cuda_memcpyHtD(" << out[0].get_name()
-                   << ", (void*)temp.data(), " << out[0].get_size() << " * "
-                   << out[0].get_element_type().size() << ");\n";
-            return;
+            // writer << out[0].get_type()
```

Comment:
remove commented out code

Reply:
Those two condition should be gone when merge with master..
```
if (args[0].get_size() == 0)
...
else if (args[0].get_size() == out[0].get_size())
```
Handled in cuda_emitter.